include/dlaf/eigensolver/bt_band_to_tridiag/impl.h

```diff
@@ -375,8 +375,8 @@ void BackTransformationT2B<B, D, T>::call(const SizeType band_size, Matrix<T, D>
 
       // Note:
       // And we use it also for computing the T factor
-      auto tile_t_full = mat_t(LocalTileIndex(ij.row(), 0));
-      auto tile_t = computeTFactor(tile_hh, tile_v, splitTile(tile_t_full, {{0, 0}, {nrefls, nrefls}}));
+      auto tile_t =
+          computeTFactor(tile_hh, tile_v, splitTileDiscard(mat_t(ij), {{0, 0}, {nrefls, nrefls}}));
```

Comment:
Still `splitTileDiscard`? Maybe you have the changes locally... Same for all the other `splitTileDiscard`s.